pom.xml

```diff
@@ -13,7 +13,6 @@
     <version>7</version>
   </parent>
 
-  <groupId>org.wildfly.swarm</groupId>
```

Comment:
The `groupId` is useful to have in order to avoid irritating the `maven-release-plugin` if I recall.  Or maybe just the `maven-version-plugin`.  But one gets confused if `groupId` is implicit and not explicit.